test/cypress/integration/widgets/deployment_actions_spec.ts

```diff
@@ -125,7 +125,9 @@ describe('Deployment Action Buttons widget', () => {
         });
 
         it('adds new label by dropdown selection', () => {
-            cy.interceptSp('GET', '/labels/deployments?_search=exist').as('fetchFilteredKeys');
+            cy.interceptSp('GET', '/labels/deployments?_sort=id&_size=10&_offset=0&_search=exist').as(
```

Comment:
Minor:

If I'm not mistaken, you could have used the RegExp pattern here to skip specifying all query parameters verbatim :smile: 

```suggestion
            cy.interceptSp('GET', /\/labels\/deployments\?(.*)_search=exist').as(
```

Reply:
I don't change this to regex to have add minimal change, but make sense to have regexp heer